cypress/e2e/09-multiplex_testing.cy.js

```diff
@@ -1,37 +1,50 @@
 import { aliasGraphqlOperations } from "../utils/graphql-test-utils";
-import { loginHooks } from "../support/e2e";
+import {
+  generateMultiplexDevice,
+  generatePatient,
+  loginHooks,
+  testNumber,
+} from "../support/e2e";
 import { graphqlURL } from "../utils/request-utils";
+import {
+  cleanUpPreviousRunSetupData,
+  cleanUpRunOktaOrgs,
+  setupMultiplexDevice,
+  setupRunData,
+  setupPatient,
+} from "../utils/setup-utils";
+
+const specRunName = "spec09";
+const currentSpecRunVersionName = `${testNumber()}-cypress-${specRunName}`;
 
 loginHooks();
 describe("Testing with multiplex devices", () => {
-  let patient, facility, multiplexDeviceName;
+  let facilityId, patientId;
+  const patient = generatePatient();
+  const multiplexDevice = generateMultiplexDevice();
 
   before(() => {
-    cy.makePOSTRequest({
-      operationName: "WhoAmI",
-      variables: {},
-      query:
-        "query WhoAmI {\n  whoami {\n organization {\n    facilities {\n      id\n      name\n      __typename\n    }\n    __typename\n  }\n} \n}",
-    }).then((res) => {
-      facility = res.body.data.whoami.organization.facilities[0];
-      cy.makePOSTRequest({
-        operationName: "GetPatientsByFacility",
-        variables: {
-          facilityId: facility.id,
-          pageNumber: 0,
-          pageSize: 1,
-          archivedStatus: "UNARCHIVED",
-        },
-        query:
-          "query GetPatientsByFacility($facilityId: ID!, $pageNumber: Int!, $pageSize: Int!, $archivedStatus: ArchivedStatus, $namePrefixMatch: String) {\n  patients(\n    facilityId: $facilityId\n    pageNumber: $pageNumber\n    pageSize: $pageSize\n    archivedStatus: $archivedStatus\n    namePrefixMatch: $namePrefixMatch\n  ) {\n    internalId\n    firstName\n    lastName\n    middleName\n    birthDate\n    isDeleted\n    role\n    lastTest {\n      dateAdded\n      __typename\n    }\n    __typename\n  }\n}",
-      }).then((res) => {
-        patient = res.body.data.patients[0];
-      });
-    });
+    cy.task("getSpecRunVersionName", specRunName).then(
+      (prevSpecRunVersionName) => {
+        if (prevSpecRunVersionName) {
+          cleanUpPreviousRunSetupData(prevSpecRunVersionName);
+          cleanUpRunOktaOrgs(prevSpecRunVersionName);
+        }
+        let data = {
+          specRunName: specRunName,
+          versionName: currentSpecRunVersionName,
+        };
+        cy.task("setSpecRunVersionName", data);
 
-    cy.task("getMultiplexDeviceName").then((name) => {
```

Comment:
because we are removing this here -- I think we can remove https://github.com/CDCgov/prime-simplereport/blob/751d08f8c807301905c867786d1189915ca1974d/cypress/cypress.config.js#L73